src/core/server/legacy/legacy_service.ts

```diff
@@ -371,7 +371,7 @@ export class LegacyService implements CoreService {
     // We only want one REPL.
     if (this.coreContext.env.cliArgs.repl && process.env.kbnWorkerType === 'server') {
       // eslint-disable-next-line @typescript-eslint/no-var-requires
-      require('../../../cli/repl').startRepl(kbnServer);
+      require('./cli').startRepl(kbnServer);
```

Comment:
a temporary hack to remove type dependencies between the `core` and `cli`. I'm going to enforce strict separation

Reply:
created issue https://github.com/elastic/kibana/issues/76935